controlplane/kubeadm/api/v1alpha3/conversion_test.go

```diff
@@ -70,3 +71,7 @@ func cabpkBootstrapTokenStringFuzzer(in *cabpkv1.BootstrapTokenString, c fuzz.Co
 	in.ID = "abcdef"
 	in.Secret = "abcdef0123456789"
 }
+
+func kubeadmBookstrapDNSFuzzer(in *kubeadmv1beta1.DNS, c fuzz.Continue) {
+	in.Type = "kube-dns"
```

Comment:
Should we call `c.FuzzNoCustom(obj)` in this function body?
Also, why are we forcing the type to "kube-dns" instead of "" used in similar funcs? 
What about adding a comment?

Reply:
Yes good catch!

The kube-dns was an error of mine (I've should have put here CoreDNS but as you suggest I'll live the empty string "")